scripts/commands/checkPagesRender.ts

```diff
@@ -83,17 +83,27 @@ zxMain(async () => {
 
   let failures: string[] = [];
   let numFilesChecked = 0;
+  let renderTimes = [];
   for (const fp of files) {
     numFilesChecked++;
-    const rendered = await canRender(fp);
-    if (!rendered) {
+    const response = await canRender(fp);
+
+    if (response.ok) {
+      renderTimes.push(response.timeTaken!);
+    } else {
       console.error(`❌ Failed to render: ${fp}`);
       failures.push(fp);
     }
 
     // This script can be slow, so log progress every 10 files.
     if (numFilesChecked % 10 == 0) {
-      console.log(`Checked ${numFilesChecked} / ${files.length} pages`);
+      const averageTime = (
+        renderTimes.reduce((a, b) => a + b, 0) / renderTimes.length
+      ).toFixed(0);
+      console.log(
+        `Checked ${numFilesChecked} / ${files.length} pages (~${averageTime}ms per page)`,
```

Comment:
Thoughts on instead computing the average for the batch of 10 files? That could help us to understand if certain files are acting up, like if we have outliers. This was indeed the problem with the static site building.

Reply:
That's what I wanted it to do, I reset `renderTimes` in the L106